android_17/src/product_data.js

```diff
@@ -1,28 +1,20 @@
-import React from 'react'
+import React from 'react';
 
-export class ProductData extends React.Component {
-    render() {
-        const productData = [
-            {
-                id: 1,
-                product_sid: '5MJBQJ',
-                name: 'apple',
-                price: '1.49'
-            },
-            {
-                id: 3,
-                product_sid: '58ZIX6',
-                name: 'strawberries',
-                price: '4.98'
-            }
-        ];
-        return (
-            <div>
-                {productData}
-            </div>
-        )
+
+const ProductData = [
```

Comment:
nit: let's either change this to all lower case, because it's a variable and not a class; or change it to all caps `PRODUCT_DATA` because it's a global variable

Reply:
Done